docs/network/architecture.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Etherlink architecture
+---
+
+Etherlink's main components are its nodes and the sequencer.
+
+## Sequencer
+
+Etherlink relies on a sequencer to process transactions.
+It receives transactions from nodes and packages them into _blueprints_, which are like blocks.
```

Comment:
Are we sure we want to mention blueprints here? I have always considered them as internal/implementation details (I don’t think I was mentioning blueprints in my draft version… but I have a doubt now)

Reply:
It seemed like an internal detail to me, too. I needed some way to describe a group of transactions; it seemed strange to talk about the sequencer finalizing single transactions. Without blueprints, is there a way to describe the point at which the transaction is final as far as the sequencer goes? I agree it's a complex detail that we could try to leave out.